src/resolvers.js

```diff
@@ -30,9 +30,9 @@ const resolvers = {
 
   // organism attribute resolvers
   Organism: {
-    genes: async (gene, { }, { dataSources }) => {
-      const organism = gene.organismId;
-      return dataSources.legumemineAPI.getGenes({organism});
+    genes: async (organism, { }, { dataSources }) => {
+      const organism_id = organism.id
+      return dataSources.legumemineAPI.getGenes(organism_id);
```

Comment:
Clearly I was copy-pasting here (gene vs organism), so thanks for fixing that! This seems to be working correctly, but the call to getGenes should use the named parameters syntax. See my other comment for why.